tools/vms-generator/utils/utils.go

```diff
@@ -629,7 +634,6 @@ func GetVMIKernelBoot() *v1.VirtualMachineInstance {
 	addKernelBootContainer(&vmi.Spec, image, KernelArgs, kernelPath, initrdPath)
 
 	vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
```

Comment:
hm, looks like it should be a part of "_func GetVMIKernelBoot()_" ??

Reply:
Thanks :) done